### PR TITLE
Improve Ability of wrap now receives multiple args

### PR DESCRIPTION
**Pull request by krishnarohitreddy**

### CODE REVIEW

File: activesupport/lib/active_support/core_ext/array/wrap.rb [hunk 1]
```diff
@@ -34,10 +36,10 @@ class Array
   #
   # The differences with <tt>Kernel#Array</tt> explained above
   # apply to the rest of <tt>object</tt>s.
-  def self.wrap(object)
+  def self.wrap(*object)
     if object.nil?
       []
-    elsif object.respond_to?(:to_ary)
+    elsif object.respond_to?(:to_ary)
```
Comment (repinel) [inline activesupport/lib/active_support/core_ext/array/wrap.rb]: :scissors: space

Comment (krishnarohitreddy) [inline activesupport/lib/active_support/core_ext/array/wrap.rb]: whops

**Comment by rafaelfranca** [comment]
Thank you for the pull request. Could you explain where it is useful? `Array.wrap` is usually called using a variable, and in that case you can have a array or a object. If you are calling `Array.wrap` like the example:

```
Array.wrap(1, 2, 3, 4)
```

You should just make it an array:

```
[1, 2, 3, 4]
```

And save the `wrap` call.

**Comment by krishnarohitreddy** [comment]
Nice point, better syntax their should be more work around to remove nil objects,

**Pull request closed without merging**